docs/tutorial/event_sourcing_basics.md

```diff
@@ -1,49 +1,48 @@
 # Event Sourcing Basics
 
-Event Sourcing focuses on business processes. The reason is simple. Every write operation in a domain model causes
-a state change. When using Event Sourcing those state changes are described by domain events. For example if an order gets payed
-this is a state change (company gets $$$) and we can describe it with an event using past tense: **OrderPayed**.
-We simply do that for all state changes. What we get is a series of domain events¹.
+Event Sourcing focuses on business processes. Every write operation in a domain model causes
+a state change, described by a domain event. For example, an order getting paid
+is a state change (company gets $$$), and we can describe it with an event using past tense: **OrderPaid**.
```

Comment:
I had to look this one up, and I don't believe it is a US vs UK spelling. I believe `payed` is only correct from a "nautical" perspective, where `paid` should be used in any other cases.